query-node/schema.graphql

```diff
@@ -237,45 +281,60 @@ type Video @entity {
   channel: Channel!
 
   "Reference to a video category"
-  category: Category!
+  category: Category
 
   "The title of the video"
-  title: String! @fulltext(query: "search")
+  title: String @fulltext(query: "search")
 
   "The description of the Video"
-  description: String!
+  description: String
 
   "Video duration in seconds"
-  duration: Int!
+  duration: Int
 
   "Video's skippable intro duration in seconds"
   skippableIntroDuration: Int
 
   "Video thumbnail url (recommended ratio: 16:9)"
-  thumbnailUrl: String!
+  thumbnailUrl: String
 
   "Video's main langauge"
   language: Language
 
-  "Reference to VideoMedia"
-  media: VideoMedia!
-
   "Whether or not Video contains marketing"
   hasMarketing: Boolean
 
   "If the Video was published on other platform before beeing published on Joystream - the original publication date"
-  publishedBeforeJoystream: Int
+  publishedBeforeJoystream: DateTime
 
   "Whether the Video is supposed to be publically displayed"
-  isPublic: Boolean!
+  isPublic: Boolean
 
-  "Video curation status set by the Curator"
-  isCurated: Boolean!
+  "Flag signaling whether a video is censored."
+  isCensored: Boolean!
 
   "Whether the Video contains explicit material."
-  isExplicit: Boolean!
+  isExplicit: Boolean
+
+  license: LicenseEntity
```

Comment:
revise the LicenseEntity type it doesn't match with our design anymore: https://github.com/Joystream/joystream/blob/sumer/content-metadata-protobuf/doc/index.md#license

Reply:
`License` has been reworked according to metadata.